client/java/src/main/java/com/linecorp/centraldogma/client/AbstractWatcher.java

```diff
@@ -124,6 +157,16 @@ void start() {
         if (state.compareAndSet(State.INIT, State.STARTED)) {
             scheduleWatch(0);
         }
+        if (meterRegistry != null) {
+            meterRegistry.gauge(LATEST_REVISION_METER_NAME, tags, this,
+                                watcher -> Optional.ofNullable(watcher.latest)
+                                                   .map(it -> it.revision().major())
+                                                   .orElse(-1));
```

Comment:
Micro optimization) Should we use `if ( != null)` instead because we prefer to minimize object allocations even though it is trivial?

Reply:
OK, fixed